packages/core/strapi/lib/commands/develop.js

```diff
@@ -120,7 +120,7 @@ function watchFileChanges({ dir, strapiInstance, watchIgnoreFiles, polling }) {
       /(^|[/\\])\../, // dot files
       /tmp/,
       '**/src/admin/**',
-      '**/src/plugins/**/admin/',
+      '**/src/plugins/**/admin/**',
       '**/documentation',
```

Comment:
Why does it work for `documentation` & `node_modules` but not for the plugins though?

Reply:
Here's the [documentation](https://github.com/micromatch/anymatch) of the library chokidar uses so I guess it was because the pattern was incorrect